message_tagging_service/consumer.py

```diff
@@ -55,3 +53,42 @@ def consume(self, msg):
             # file is empty, catch this case and skip to handle the tag.
             if rule_defs is not None:
                 tagging_service.handle(rule_defs, event_msg)
```

Comment:
How about publishing a message that build was handled and a tagging request was/wasn't sent? If tagging was requested let's also include the task ID.

Reply:
Hi @lcarva , sorry, I don't understand your question. Can you please repeat it again?

Reply:
After re-reading what I wrote I understand the confusion :D

I think MTS should publish a message whenever it processes a module build, regardless if a tag was applied or not. For example, module build X goes into "ready" state, MTS decides to tag it as "foo", then MTS should send a message that this action was taken and that the tagging operation can be tracked by koji taskID 123. If MTS decides that no tag should be applied, it should also send a message indicating this behavior.

This will make it much easier to trace what MTS is doing.